Documentation/docker/index.rst

```diff
@@ -4,28 +4,27 @@ Docker
 
 Cilium can be integrated with Docker in two ways:
 
-* via the `CNI` interface. This is the method chosen by `Kubernetes` and :ref:`Mesos`.
-* via Docker's `libnetwork`_ plugin interface if networking is to be managed by
-  the Docker runtime. This method is used for example by Docker compose.
-
-When using Cilium with Docker's libnetwork, one creates a single logical Docker
-network of type ``cilium`` and with an IPAM-driver of type ``cilium``, which
-delegates control over IP address management and network connectivity to Cilium
-for all containers attached to this network for both IPv4 and IPv6
-connectivity.  Each Docker container gets an IP address from the node prefix of
-the node running the container.
-
-When deployed with Docker, each Linux node runs a ``cilium-docker`` agent,
-which receives libnetwork calls from Docker and then communicates with the
+* via the `CNI` interface. This method is used by `Kubernetes` and :ref:`Mesos`.
+* via Docker's `libnetwork`_ plugin interface, if networking is to be managed by
+  the Docker runtime. This method is used, for example, by Docker compose.
```

Comment:
Docker compose --> Docker Compose . Maybe link to Docker Compose?

Reply:
I linked to the docs but I can link to the github repo if you prefer. I also fixed the capitalisation in the docker compose gsg.